Education section added

Ergouzii opened the pull request:
As discussed in Issue #246 , I created this PR and the changes can be previewed [here](https://developer-folio-with-education.netlify.app/).

Resolves #246.

kartikcho commented:
Hi, thanks for working on this!

I haven't taken a full look at the code yet. I think we can add some interactivity to the cards by changing the `border-bottom-color` to black or our 'brand' purple in the `.education-card` class. 
Maybe we can pair it with an expanding transition if possible or any other minimal one to make the component more interesting.

Let me know what you think about adding this @Ergouzii!

kartikcho commented:
I edited your original PR description to add a closure comment.
It will auto close the issue your PR is addressing when it's merged :)

Ergouzii commented:
@kartikcho Great points! Just changed the border's color and added a sliding effect. Please check out [the site](https://developer-folio-with-education.netlify.app/) and let me know :)

kartikcho commented:
I'm sorry if I wasn't clear earlier, I meant we can add hover effects like changing border color or a transition on hover. The purple looks great, just transition from the earlier grey bottom border to purple bottom border on card hover. Everything else looks cool!

Ergouzii commented:
@kartikcho Haha sorry for that, just updated the effect. The border now will change to purple and be extended on hover. Please check and let me know!

kartikcho commented:
This looks really neat!
Everything is set for a review but I ran into some small responsiveness bugs.

The class `.education-card-right` has `max-width: 70%;` but you've not set a media query to expand it to 100% for smaller screens which cause this issue,

![image](https://user-images.githubusercontent.com/48270786/94105128-39e25c00-fe56-11ea-8e8d-dfd6a9796ced.png)

Same for the bottom border, should set it 100% at the same width query at which it gets center aligned.

Ergouzii commented:
@kartikcho Good catch! Just fixed it.

kartikcho reviewed (changes requested):
Great PR! Just a few nits to fix and we can get this merged. Awesome work!

Ergouzii commented on src/containers/education/Education.css:
```diff
@@ -0,0 +1,46 @@
+.education-heading {
+  font-size: 56px;
```
Sorry I misclicked "Resolve Conversation" on your comment for this code block. Yes I set it to 56px to be consistent with other sections. For example, `Skills.css` has a heading of 56px on large screens, 40px on medium screens, and 30px on small screens. I made this education heading the same way.

Ergouzii replied:
@kartikcho

kartikcho replied:
Ah I see, it's alright then. 
Also, you can unresolve a comment if you accidentally clicked on it so don't worry about it. As a general good practice you should mark the comments addressed in a follow up commit as resolved.

Ergouzii commented on src/portfolio.js:
```diff
@@ -97,6 +97,36 @@ https://fontawesome.com/icons?d=gallery */
+    {
+      schoolName: "Harvard University",
+      logo: require("./assets/images/harvardLogo.png"),
+      degree: "Master of Science in Computer Science",
```
Changed to `subHeader`

Ergouzii commented on src/portfolio.js:
```diff
@@ -97,6 +97,36 @@ https://fontawesome.com/icons?d=gallery */
+
+const educationInfo = {
+  viewEducation: true, // Set it to true to see education section
+  title: "Education",
```
Deleted `title` here

Ergouzii commented on src/containers/education/Education.js:
```diff
@@ -0,0 +1,19 @@
+  return (
+    <div className="education-main-div" id="education">
+      <div>
+        <h1 className="education-heading">{educationInfo.title}</h1>
```
Done

Ergouzii commented on src/containers/education/Education.css:
```diff
@@ -0,0 +1,46 @@
+  font-weight: 400;
+}
+
+.education-main-div {
```
Done

Ergouzii commented on src/containers/education/Education.css:
```diff
@@ -0,0 +1,46 @@
+  margin-bottom: 30px;
+}
+
+.education-card-div {
```
`education-card-div` is the container for all cards, and the name `education-card` is already been used by each individual card. Changed it to `education-cards`

Ergouzii commented on src/components/header/Header.js:
```diff
@@ -1,7 +1,7 @@
+import {greeting, educationInfo, workExperiences} from "../../portfolio";
 
 function Header() {
   const exp = workExperiences.viewExperiences;
```
Done

Ergouzii commented on src/components/educationCard/EducationCard.css:
```diff
@@ -0,0 +1,140 @@
+.education-card:hover + .education-card-border {
+  border-color: #a9a7f9;
+  width: 100%;
+  -webkit-transition: width 1s ease-in-out;
```
Changed to 0.6s

Ergouzii commented on src/components/educationCard/EducationCard.css:
```diff
@@ -0,0 +1,140 @@
+  max-width: 70%;
+}
+
+.education-text-degree {
```
Changed to `subHeader`

kartikcho commented on src/components/educationCard/EducationCard.css:
```diff
@@ -0,0 +1,140 @@
+  flex-direction: row;
+}
+
+.education-div-school {
```
Avoid using element names in classes like you've done here. Instead go for smaller descriptive names like `education-school-container` or something

kartikcho commented on src/containers/education/Education.css:
```diff
@@ -0,0 +1,46 @@
+  margin-bottom: 30px;
+}
+
+.education-cards {
```
My bad earlier, to make it a bit different, we can go for `education-card-container` or something of the sort. What do you think?

kartikcho commented on src/containers/education/Education.js:
```diff
@@ -0,0 +1,19 @@
+export default function Education() {
+  return (
+    <div className="education-section" id="education">
+      <div>
```
Any specific reason for this empty div?

kartikcho reviewed (approved):
Just one last round of changes left to address and we're good!

Ergouzii commented:
@kartikcho Done the last round of fixes :)

kartikcho reviewed (approved):
Everything looks great, LGTM!

kartikcho commented:
@all-contributors add @Ergouzii for code and design.
